src/resolve-config.js

```diff
@@ -3,10 +3,23 @@
 const cosmiconfig = require("cosmiconfig");
 const minimatch = require("minimatch");
 
-const asyncWithCache = cosmiconfig("prettier");
-const asyncNoCache = cosmiconfig("prettier", { cache: false });
-const syncWithCache = cosmiconfig("prettier", { sync: true });
-const syncNoCache = cosmiconfig("prettier", { cache: false, sync: true });
+const cosmiconfigBaseOptions = {
+  rcExtensions: true
+};
+
+function getCosmiconfigWithOptions(options = {}) {
```

Comment:
Unfortunately we cannot use default values because we support node v4

Reply:
I was looking for which version is supported, but didn't find it in package.json or the docs. Will fix.

Reply:
Maybe we should add an `engines` field to package.json: https://docs.npmjs.com/files/package.json#engines

Reply:
@josephfrazier added `"node": ">=4"` to `package.json` :)